### ISSUE TITLE
Pontoon commits .properties strings with line break as junk

### ISSUE THREAD

**Issue by bugzilla-to-github** [mozilla/pontoon#2207]
*This issue was created automatically by a [script](https://github.com/mathjazz/bugzilla2github/).*
## [Bug 1611667](https://bugzilla.mozilla.org/show_bug.cgi?id=1611667)
Bug Reporter: @flodolo
CC:   @Pike, @Pike
Depends on:   [Bug 1696486](https://bugzilla.mozilla.org/show_bug.cgi?id=1696486)

It happened at least twice in the last week: translation in a .properties file including a line break is serialized as is, resulting in junk.

https://hg.mozilla.org/l10n-central/hy-AM/rev/6552dfee38b49517c83a31ac2fd888cf4bf3056a

The new line should be replaced by a `\`

**Comment by mathjazz** [comment]
Looks like this still needs to be fixed?
https://github.com/mozilla-l10n/pontoon-test/commit/3239d50afd4feea105e98d13f8f21b6eaad1a33c

@eemeli ?

**Comment by eemeli** [comment]
Yeah, the current serialization relies on the value stored in the database being safe to serialize, which isn't true for .properties messages with newlines. There should be a `.replace('\n', '\\n')` done on the Pontoon side for this.

**Comment by flodolo** [comment]
Actually, let's assume this was verified and it's a regression.
